wsi/drm/drmbuffer.cpp

```diff
@@ -286,7 +286,7 @@ const MediaResourceHandle& DrmBuffer::GetMediaResource(MediaDisplay display,
   external.width = width_;
   external.height = height_;
   external.num_planes = total_planes_;
-  unsigned long prime_fd = prime_fd_;
+  uintptr_t prime_fd = prime_fd_;
```

Comment:
We will have cases where we have some platforms using libva <2.0 and some 2.0 and greater. We should have a compile time check for this.